src/shared/highlighted-field/index.js

```diff
@@ -0,0 +1,4 @@
+export * from './highlighted-field-context.js'
+export { default as HighlightedFieldIdsProvider } from './highlighted-field-ids-provider.js'
+export * from './use-highlighted-field-context.js'
+export { default as useCreateHighlightedFieldIds } from './use-highlighted-field.js'
```

Comment:
Why is this name different from the `default` function name? `useHighlightedFieldId`?